hw/ip/otbn/doc/_index.md

```diff
@@ -519,6 +519,45 @@ The `URND` LFSR is seeded once from the EDN connected via `edn_urnd` when OTBN s
 Each new execution of OTBN will reseed the `URND` LFSR.
 The LFSR state is advanced every cycle when OTBN is running.
 
+### Operational States {#design-details-operational-states}
+
+OTBN can be in operational states: busy performing an operation, or idle.
```

Comment:
I updated the sentence. I kept the "busy" category without listing the states explicitly to avoid duplication of this information; it's rather likely we'll add more commands as we go forward, and history shows we always forget updating one of the duplicates when doing so.

Background: This sentence got mangled when I removed the LOCKED state, which is the terminal state after we've detected a fatal error. I'll bring that state back in a separate PR, but stripped it out of this one to keep it "minimal."

I agree with the state diagram, but I'd prefer doing it after we have introduced the LOCKED state to avoid redoing it in a couple days. I filed https://github.com/lowRISC/opentitan/issues/7900 to keep track of it.

Reply:
I've now opened https://github.com/lowRISC/opentitan/pull/7904 to introduce the LOCKED state.